code/DepMapGeneEss/PlotGeneEss.R

```diff
@@ -0,0 +1,62 @@
+library(ggplot2)
+library(tidyverse)
+
+setwd("C:/Work/MatlabCode/components/human-GEM/Human-GEMDepMapEval/Human-GEM/code/DepMapGeneEss") #Modify to match your file system
```

Comment:
It would be much nicer here with a relative path.

Reply:
Agreed, but the same problem

Reply:
I've left [a separate suggestion](https://github.com/SysBioChalmers/Human-GEM/pull/574/files#r1201489503) for how this could be achieved.

Reply:
Should the same suggestion be applied here?
```suggestion
setwd(dirname(rstudioapi::getActiveDocumentContext()$path))
```